fixture.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
+const {app} = require('electron');
 const fetch = require('.');
 
+app.on('window-all-closed', () => {});
```

Comment:
Why this?

Reply:
It was already there in the proof of concept and I just tought it was required, I was obviously wrong.